voice-assistant/esp32-s3-box-3.yaml

```diff
@@ -149,65 +159,77 @@ voice_assistant:
   on_listening:
     - display.page.show: listening_page
     - component.update: s3_box_lcd
-  on_tts_start:
-    - display.page.show: talking_page
-    - component.update: s3_box_lcd
-  on_end:
-    - delay: 100ms
-    - wait_until:
-        not:
-          speaker.is_playing:
-    - script.execute: reset_display
   on_stt_vad_end:
     - display.page.show: thinking_page
     - component.update: s3_box_lcd
-  on_error:
-    - display.page.show: error_page
+  on_tts_stream_start:
+    - display.page.show: replying_page
     - component.update: s3_box_lcd
-    - delay: 1s
+  on_tts_stream_end:
     - script.execute: reset_display
+  on_error:
+    - if:
+        condition:
+          lambda: return !id(init_in_progress);
+        then:  
+          - display.page.show: error_page
+          - component.update: s3_box_lcd
+          - delay: 1s
+          - script.execute: reset_display
   on_client_connected:
     - if:
         condition:
           switch.is_on: use_wake_word
         then:
           - voice_assistant.start_continuous:
+          - lambda: id(init_in_progress) = false;
           - script.execute: reset_display
   on_client_disconnected:
     - if:
         condition:
           switch.is_on: use_wake_word
         then:
           - voice_assistant.stop:
-          - display.page.show: sleeping_page
+          - display.page.show: no_pipeline_page
           - component.update: s3_box_lcd
 
 script:
   - id: reset_display
     then:
       - if:
           condition:
-            wifi.connected:
+            lambda: return !id(init_in_progress);
           then:
             - if:
                 condition:
-                  api.connected:
+                  wifi.connected:
                 then:
                   - if:
                       condition:
-                        switch.is_on: use_wake_word
+                        api.connected:
                       then:
-                        - display.page.show: idle_page
-                        - component.update: s3_box_lcd
+#                        - if:
+#                            condition:
+#                              voice_assistant.connected:
+#                            then:
```

Comment:
Its better to just delete lines instead of commenting them. They can easily be found in the PR/commit history
```suggestion
```

Reply:
Actually, these are brand new lines 😊 Waiting for the `voice_assistant.connected:` to be merged.
From what I understand. It is the case now. So I guess I can remove them!